### DISCUSSION TITLE
flyTo and fitBounds paddings add up

### DISCUSSION THREAD

**Discussion by Varixo** [maplibre/maplibre-gl-js#2984] [Q&A] [answered]
Hey, I ran into a similar issue as in mapbox-gl:
https://github.com/mapbox/mapbox-gl-js/issues/11831

There is any "normal" solution for this?

**Comment by HarelM** [comment, marked as answer]
The linked thread does a very nice job in explaining the problem and the possible solutions...

**Reply by Varixo** [reply]
yes, but I feel like it's a very hacky solution and I was wondering if there is any better solution than described there

**Reply by HarelM** [reply]
Probably not. I'm not sure why setting padding explicitly is a hacky solution though...

**Reply by Varixo** [reply]
Setting padding immediately after `flyTo` is somehow breaking the method and doesn't work at all. Anyways thanks, I'll try some things. I mean, maybe I need to wait for the 'moveend' event to fire 
(like `map.once('moveend', () => map.setPadding({top: 0, bottom: 0, left: 0, right: 0}));` or something like that)
